src/walinfo.c

```diff
@@ -60,20 +66,27 @@ usage(void)
    printf("  pgmoneta-walinfo <file>\n");
    printf("\n");
    printf("Options:\n");
-   printf("  -c, --config CONFIG_FILE Set the path to the pgmoneta.conf file\n");
-   printf("  -o, --output FILE        Output file\n");
-   printf("  -F, --format             Output format (raw, json)\n");
-   printf("  -L, --logfile FILE       Set the log file\n");
-   printf("  -q, --quiet              No output only result\n");
-   printf("      --color              Use colors (on, off)\n");
-   printf("  -r, --rmgr               Filter on a resource manager\n");
-   printf("  -s, --start              Filter on a start LSN\n");
-   printf("  -e, --end                Filter on an end LSN\n");
-   printf("  -x, --xid                Filter on an XID\n");
-   printf("  -l, --limit              Limit number of outputs\n");
-   printf("  -v, --verbose            Output result\n");
-   printf("  -V, --version            Display version information\n");
-   printf("  -?, --help               Display help\n");
+   printf("  -c,   --config CONFIG_FILE                     Set the path to the pgmoneta_walinfo.conf file\n");
+   printf("  -u,   --users USERS_FILE                       Set the path to the pgmoneta_users.conf file \n");
+   printf("  -RT, --tablespaces TBLSPCS,COMMA,SEPARATED     Filter on tablspaces\n");
```

Comment:
You don't need `TBLSPCS,COMMA,SEPARATED` - that should be explained in the manuals